### DISCUSSION TITLE
dunst - i3 - ubuntu 22.04 -  origin option unavailable

### DISCUSSION THREAD

**Discussion by giuseppe-chiari** [dunst-project/dunst#1272] [Q&A] [answered]
Recently moved from Fedora 38 to Ubuntu 22.04.
I used to have my dunst notifications in the bottom-left of the monitor through the `origin` option of dunst. However such option looks like is not available on Ubuntu (I cannot find it in `man dunst`) is there a way to achieve the same behavior also in Ubuntu?
Or am I missing something and that option should be available also on Ubuntu?

**Comment by Narrat** [comment]
Did you look at the man-page for the configuration? `man 5 dunst`
`origin` is mentioned there.
But in general the option should be there. What may happen is that a different program grabs the hint for displaying notifications. ( https://github.com/dunst-project/dunst/discussions/1267 )

**Reply by giuseppe-chiari** [reply]
The output of `man 5 dunst` is the folllowing:

`No manual entry for dunst in section 5`

I've noticed that I have version 1.5.0 (2020-07-23) of dunst, which seems to be the newest available according to `sudo apt install dunst`

**Reply by Narrat** [reply]
Threw yesterday a symbolic coin on what I want to go with: Version to old or wrong notifier...
Good thing I don't gamble. But wouldn't have thought there is a version this old still around.
`1.5` doesn't have `origin`, as you discovered. Not sure anymore if it could be done with the geometry option.

On how to proceed... you could manually compile a newer release or maybe there is a PPA (if you want to trust such a private repo)

**Reply by Narrat** [reply]
Or maybe it is possible with `geometry`
To quote from the `1.7.0` release: 
> geometry has been replaced by origin, width, height, offset and
notification_height. This allows for more flexible geometry settings. (https://github.com/dunst-project/dunst/pull/855)

**Reply by giuseppe-chiari** [reply]
I think I'll go with the compile it myself option. I will update this post if that solves the problem.
Crazy, though, that the most updated version that `apt` can get, on Ubuntu 22.04, is that old.

**Reply by giuseppe-chiari** [reply, marked as answer]
Can confirm that compiling it from the GitHub repo solves the issue.
